xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/resources/ApplicationResources.properties

```diff
@@ -106,4 +106,13 @@ livedata.panel.sort.direction.descending=Descending
 livedata.panel.sort.add=Add sort
 livedata.panel.sort.delete=Delete sort
 
-livedata.displayer.link.noValue=(no value)
\ No newline at end of file
+livedata.displayer.link.noValue=(no value)
+
+livedata.displayer.boolean.true=True
+livedata.displayer.boolean.false=False
+
+livedata.displayer.xObjectProperty.missingDocumentName.errorMessage=Can't save an XObject property with a missing document name.
+livedata.displayer.xObjectProperty.failedToRetrieveField.errorMessage=Failed to retrieve the {mode} field.
+
+livedata.filter.boolean.yes=Yes
+livedata.filter.boolean.no=No
```

Comment:
Shouldn't this be consistent with the boolean displayer? If I see ``true`` and ``false`` as values in the live data then I expect to filter by ``true`` and ``false``. So we should either use ``true`` / ``false`` everywhere (displayer and filter) or ``yes`` / ``no``.